app/src/main/java/com/github/se/polyfit/ui/screen/MapScreen.kt

```diff
@@ -1,21 +1,22 @@
 package com.github.se.polyfit.ui.screen
 
 // import the colors PrimaryPink
-import android.util.Log
+
```

Comment:
I know that you wanted to group everything related to the map in a single file, but it feels a bit too clustered. I would recommend breaking it down into smaller components.